cargo 0.73 build broken on windows

I updated cargo to 0.73 from crates.io in cargo-udeps ([commit](https://github.com/est31/cargo-udeps/commit/17f086096d69ee8d03d69bba91861007d8cf9e34)) and got the following [build failure on ci](https://github.com/est31/cargo-udeps/actions/runs/5980865638/job/16227804165):

```
 error[E0432]: unresolved import `windows_sys::Win32::Storage::FileSystem::LockFileEx`
   --> C:\Users\runneradmin\.cargo\registry\src\index.crates.io-6f17d22bba15001f\cargo-0.73.0\src/cargo\util\flock.rs:443:9
    |
443 |         LockFileEx, UnlockFile, LOCKFILE_EXCLUSIVE_LOCK, LOCKFILE_FAIL_IMMEDIATELY,
    |         ^^^^^^^^^^
    |         |
    |         no `LockFileEx` in `Windows::Win32::Storage::FileSystem`
    |         help: a similar name exists in the module: `LockFile`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `cargo` (lib) due to previous error
```

If you look at the [rustdoc for LockFileEx](https://docs.rs/windows-sys/0.48.0/windows_sys/Win32/Storage/FileSystem/fn.LockFileEx.html), it writes:

```
Required features: "Win32_Storage_FileSystem", "Win32_Foundation", "Win32_System_IO"
```

Compare this with the Cargo.toml of 0.73:

https://github.com/rust-lang/cargo/blob/103a7ff2ee7678d34f34d778614c5eb2525ae9de/Cargo.toml#L180-L188

The former two are found while `Win32_System_IO` isn't present. I suppose this is the cause for the failure: in the `cargo` repo this works fine because of dependency unification and another crate in the DAG enabling the `Win32_System_IO` features. But if you use `cargo` from crates.io, then it is not enabled, which is what I ran into.

The fix is simple, just add `Win32_System_IO` to the list. Ideally this would also be published in a new release.

I have worked around the issue by making cargo-udeps depend on windows-sys and enabling the feature ([commit](https://github.com/est31/cargo-udeps/commit/51e2285425a5dcbe68c4b4ef8d0f2c5026da59bb)).

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -194,10 +194,12 @@ fwdansi.workspace = true
 workspace = true
 features = [
   "Win32_Foundation",
+  "Win32_Security",
   "Win32_Storage_FileSystem",
+  "Win32_System_IO",
   "Win32_System_Console",
-  "Win32_System_Threading",
   "Win32_System_JobObjects",
+  "Win32_System_Threading",
 ]
 
 [dev-dependencies]

